src/sync/installation.ts

```diff
@@ -43,7 +43,8 @@ interface TaskProcessors {
 		jiraHost: string,
 		repository: Repository,
 		cursor?: string | number,
-		perPage?: number
+		perPage?: number,
+		messagePayload?: Record<string, any>
```

Comment:
Don't we have types for these message payloads?  Pretty sure the queues use a generic, but there's no reason we can't do the same here and let each processor have their own typings to make it a bit safer

Reply:
Used correct types